content/post/articles/music-while-coding-is-it-worth.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Music while coding is it worth?"
+date: 2022-02-18T20:08:48+01:00
+images: ["/shaba-rex.png"]
+categories: ["Software development"]
+tags: ["Coding","Music","Music while coding","Silence","Software development","CaParleDev"]
+authors: ["Shaba-Rex"]
+---
+{{<figure src="/music-while-coding-is-it-worth/Listen-to-music.png"  alt="Liste to music while coding">}}
+
+Music is the art of arranging sounds in time through the elements of melody, harmony, rhythm for years human beings have punctuated their activities to make them more playful, more festive, more serious, whether in a football stadium animated by supporters or at parties to accompany the dance or during wedding ceremonies when the bride enters the church, in these different activities it has always been able to find her place.
```

Comment:
Love it! But we need more punctuations. Can you add it please

Reply:
Hello, my friend thank you for your review can you help me to correct it because, concerning the punctuations, I use two grammar and vocabulary checkers to help me correct my English.
Grammarly(https://app.grammarly.com) and Writer(https://writer.com)
On the first, I have two punctuation errors but, I corrected my text the two errors do not go away and, the second tells me 'no errors'

Reply:
Hello, it is more about helping to understand. For example, when you read "rhytm for human years", something is missing. Just try to make short sentences with . and , and everything will be fine.